nexus/lib/qmcpack_input.py

```diff
@@ -4841,7 +4948,7 @@ def generate_determinantset_old(type           = 'bspline',
                 QmcpackInput.class_error('excitation with vb-cb band format works only with special k-points')
             #end if
             
-            vb = int(det.size / abs(linalg.det(tilematrix))) -1  # Separate for each spin channel
+            vb = int(sdet.size / abs(linalg.sdet(tilematrix))) -1  # Separate for each spin channel
```

Comment:
This looks like overuse of find/replace.  `linalg.sdet` needs to remain `linalg.det` to calculate the determinant of the tiling matrix.

Reply:
Resolved.

Reply:
Good catch

Reply:
I'm not seeing the change here.  Committed and pushed?

Reply:
My change was pushed in commit 1153c8d with message "Fix sdet typo".
Though I kept `sdet.size`. I think only the `linalg.sdet` should be fixed, right?

Reply:
Yes